pom.xml

```diff
@@ -84,8 +84,8 @@
 
   <properties>
     <!-- language versions -->
-    <scala.version>2.10.5</scala.version>
-    <scala.binary.version>2.10</scala.binary.version>
+    <scala.version>2.11.7</scala.version>
```

Comment:
2.11.8? that's the one used by spark, https://github.com/apache/spark/blob/master/pom.xml#L158